db/migrations/02_add_time_to_peeps.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD column time TIMESTAMP;
```

Comment:
May have been easier/cleaner to just include this in the 01 migration.

Reply:
But of course, is probably as you just did it as you were going along so ignore this comment haha!